scipy/sparse/linalg/_dsolve/linsolve.py

```diff
@@ -26,6 +26,7 @@
 
 
 class MatrixRankWarning(UserWarning):
+    """Warning on exactly singular matrices."""
```

Comment:
Can you change both instances to the following 
```suggestion
    """Warning for exactly singular matrices."""
```